From 7e1e505ca90a3d89e991815e96313aa49390152f Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 15 Jul 2009 03:41:28 +0200
Subject: [PATCH 3/7] Make sure that we use "cmd" as a shell

The original idea of the MinGW32 build was probably to run the beast
from cmd.exe.  As we run it from bash now, let's make sure that "make"
does not try to execute the "copy" and "del" calls with /bin/sh, but
rather with "cmd".

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>

diff --git a/ms/mingw32.bat b/ms/mingw32.bat
index f9377a8..f85305e 100644
--- a/ms/mingw32.bat
+++ b/ms/mingw32.bat
@@ -79,7 +79,7 @@ if errorlevel 1 goto end
 rem copy ms\tlhelp32.h outinc
 
 echo Building the libraries
-mingw32-make -f ms/mingw32a.mak
+mingw32-make -f ms/mingw32a.mak SHELL=cmd
 if errorlevel 1 goto end
 
 echo Generating the DLLs and input libraries
-- 
2.1.0

